scss/forms/_questions.scss

```diff
@@ -0,0 +1,29 @@
+@import "_colours";
+@import "_typography.scss";
+
+.question {
+  margin: 15px 0 0 0;
+  clear: both;
+}
+
+.first-question {
+  margin-top: 0;
+}
```

Comment:
I've amended the naming for consistency. I think the use of `@extend` should be implemented here but the same should be done to other parts of the CSS so I'm going to do that in a separate pull request.